python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -270,6 +271,50 @@ def build(
         )
         return feature_view
 
+    def get_step_size(self) -> timedelta:
+        """
+        Returns the step size of the sliding window features in the feature view. All
+        the sliding window features must have the same step size.
+        """
+        step_size_set = set(
+            [
+                feature.transform.step_size
+                for feature in self.get_resolved_features()
```

Comment:
It seems that getting the step size of a sliding feature view does not necessarily require that the feature view is resolved, as it does not rely on source table's properties, nor would Feathub convert any `str` feature to sliding window transform.